mvp-0/kaas/.env

```diff
@@ -0,0 +1,7 @@
+# SQLITE_DATABASE_PATH="/tmp/kaas.db"
```

Comment:
this should probably be something like:
```suggestion
# SQLITE_DATABASE_PATH="/var/run/kaas/kaas.db"
```
but we need to make sure the directory exists.

Reply:
I was not able to apply this suggestion, (probably because the dir structure has changed in the previous commit), so instead I incorporated your suggestion as a fixup: 18139afc76d6563037271d96c79597b8ed743cbc 
Thanks!  

> but we need to make sure the directory exists.

IMO it is not necessary as this is only an example (commented out) of how you can override the default configuration